### DISCUSSION TITLE
Achievement Guide/Road Maps

### DISCUSSION THREAD

**Discussion by grenade44** [RetroAchievements/RAWeb#662] [Ideas]
To have the ability to write a road map/achievement guide to help users unlock all the achievements. Just like x360a.org does [https://www.xboxachievements.com/game/aca-neogeo-metal-slug-2/guide/](https://www.xboxachievements.com/game/aca-neogeo-metal-slug-2/guide/)

**Comment by kdecks** [comment]
Subpages from the game page perhaps. Maybe a forum topic with a single author? Perhaps a modified forum group just for these.

Multiple guides could be submitted.

**Comment by grenade44** [comment]
Subpage from gamepage I think would be good, that way you could also have a master page that lists all the games with road maps, which might help users decide what game to play next if there’s a road map.

**Comment by meleu** [comment]
IMHO our cheevos are a bit "dynamic". I mean not "solid" enough for this.

**Comment by grenade44** [comment]
Just came across this forum post about people wanting guides
http://retroachievements.org/viewtopic.php?t=5798&c=
And even maguspc starting a site just for guides http://raguides.iblogger.org/doku.php (still a working progress)

**Comment by enzyme-1** [comment]
This feature can help with the frustration some players have with achievements, some of the things I've got and seen in the forums and game pages recently:
"I need more explanation of how this achievement works", "this did not unlock, am I doing this correctly?", "how do I do this? Is there a video online of it somewhere?"

**Comment by enzyme-1** [comment]
Sometimes you can't even fit an entire description of what to do on an achievement. Sure we could just create the game topic and just post everything there, but it would be more formal and appealing to create guides. I would even be interested in making PDF versions of my guides for games that I've worked on, but if this system gets implemented on the site that would make it easier for everyone.

**Comment by meleu** [comment]
We plan to add this feature in the site at some point. But, as we don't have an ETA for this feature and I'm seeing many devs talking about guides lately, I've created the RAGuides repository: https://github.com/RetroAchievements/guides

You guys can use the repo's wiki section to add the guides you want. I've already added the [Robin Hood (NES) guide](https://github.com/RetroAchievements/guides/wiki/Robin-Hood:-Prince-of-Thieves-(NES)) there (by @ikki5) just to start with something.

Later (as we get more guides ready) I can setup an environment just like the [RADocs](http://docs.retroachievements.org) where each section/tab can be a console. It's a relatively fast thing to setup.

**Comment by enzyme-1** [comment]
Ill add one more point on why RA needs it, I kinda hinted on this back in the forum post. A few players seem to play games when they are frustrated, and log on angry, lol. Us long-time developers take a good beating from people who are not used to the open-community/user-contribution format of the site. 

The guides can help promote a more comfortable/complete experience, so those hunting for achievements and badges can enjoy the entire experience and relax a little more. I think a good guide reinforces the fun parts of the game, not just offer tips on difficulty. 

I started working on my first guide for [Blaster Master](http://retroachievements.org/game/1459). It is known to be a very hard game which suffers from not having save memory or passwords. There are games I don't know well or never played and instead of searching the net for a manual, somebody who knows the game and how the achievements function can submit a guide - that will bring more players to more titles as a result.

**Comment by enzyme-1** [comment]
OK, I finished my first guide offline, but I don't see the "New Page Button" the [README.md](https://github.com/RetroAchievements/guides/blob/master/README.md) says I'm supposed to click on. All I see is "New Repository" which doesn't seem right. Should I just wait until this feature gets implemented on RA, or is there a simple way of creating the guides as pages?

**Comment by meleu** [comment]
Looks like you're missing the step 2: 

2. Go to [the wiki section of the repository](https://github.com/RetroAchievements/guides/wiki).

**Comment by enzyme-1** [comment]
Yeah I just read that several times, did what it said, but I'm not seeing "**_New Page Button_**". Somehow I ended up "forking" guides and when I click on the "wiki" tab it now says "create new page", so something doesn't seem right here (but I don't mind doing the guides offline for now and just waiting for the official support on the site.)

**Comment by meleu** [comment]
My fault, sorry. There was a missing config restricting wiki editing. Now it's solved.

Link to go directly to the wiki: https://github.com/RetroAchievements/guides/wiki

![guides-step1](https://user-images.githubusercontent.com/8508804/43437456-ae5656b2-9478-11e8-9f30-f5ee259f78a7.png)

![guides-step2](https://user-images.githubusercontent.com/8508804/43437457-ae8f9e86-9478-11e8-8a05-1365a361d25f.png)

**Comment by enzyme-1** [comment]
Ah, now I see it! I just clicked on that and it works, thanks!

**Comment by wescopeland** [comment]
Superseded by https://github.com/RetroAchievements/RAWeb/discussions/4196.
